charts/posthog/values.yaml

```diff
@@ -271,6 +271,8 @@ ingress:
   nginx:
     # -- Whether nginx is enabled
     enabled: true
+  # -- Whether to redirect to TLS. Defaults to true.
+  redirectToTLS:
```

Comment:
Why do this in such a roundabout way (needing a custom helper)? 

Why not set it to true here and remove the code in _helpers?

Reply:
Good point, will change it.

Reply:
left it in helpers template atm (instead of changing ingress.yaml to use .Values), we might want to have more complex logic later

Reply:
We can always refactor that easily. YAGNI